Tests/Unit/DependencyInjection/DoctrinePHPCRExtensionTest.php

```diff
@@ -74,7 +73,7 @@ public function testCustomManagerRegistryService()
             'manager_registry_service_id' => 'my_phpcr_registry',
         ));
 
-        /** @var $repositoryFactory DefinitionDecorator */
+        /* @var $repositoryFactory DefinitionDecorator */
```

Comment:
this is intentionally a doc comment. tools like phpstorm use this to autocomplete and highlight errors. it should be allowed by the code style imho